packages/flutter/lib/src/material/app.dart

```diff
@@ -732,9 +742,6 @@ class MaterialApp extends StatefulWidget {
   ///  * <https://material.io/design/layout/spacing-methods.html>
   final bool debugShowMaterialGrid;
 
-  /// {@macro flutter.widgets.widgetsApp.useInheritedMediaQuery}
-  final bool useInheritedMediaQuery;
```

Comment:
Doesn't this need to be deprecated, but just not referenced in the implementation? If someone was reading it, this would break them.

Reply:
More deprecation notices added. (I would argue that technically just removing these was in line with our breaking change policy sine the removal didn't break any customer tests, but it might save some untested customers some trouble...)

Reply:
Yeah, and people are actually pretty unlikely to be accessing these in the first place, but I don't see any harm in keeping them as long as they're deprecated and not used in the implementation.

Reply:
Right, I added them back in.